server/utils/image_cropper.py

```diff
@@ -0,0 +1,16 @@
+from adapters.object_detection import DetectedObject
+from PIL import Image
+
+
+class ImageCropper:
+    def __init__(self, image: Image.Image):
+        self.image = image
+
+    def get_image_crops(self, detected_objects: list[DetectedObject]) -> list[Image.Image]:
+        cropped_elements = []
+        for detected_object in detected_objects:
+            bbox = detected_object.bbox
+            crop = (bbox.x1, bbox.y1, bbox.x2, bbox.y2)
+            cropped_element = self.image.crop(crop)
```

Comment:
`crop` method receives a tuple of ints according to docs: https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.crop

But here it is receiving a tuple of ints...
```suggestion
            crop = (int(bbox.x1), int(bbox.y1), int(bbox.x2), int(bbox.y2))
            cropped_element = self.image.crop(crop)
```

Reply:
I just changed it. Thanks for the suggestion.